.github/workflows/ci.yml

```
@@ -32,7 +32,17 @@ jobs:
 
+      - name: Hydrate the PICO sdk
+        description: Hydration is required to build the PICO tool
+        if: steps.cache-toolchain.outputs.cache-hit != 'true'
+        run: make pico_sdk
+
+      - name: Build pico tool and install
+        if: steps.cache-toolchain.outputs.cache-hit != 'true'
+        run: make picotool_install
+
       - name: Hydrate configuration
+        description: Load the configuration sub-module as it is needed to ensure the build target list is successful
```

actionlint (1.7.7)

unexpected key "description" for "step" section. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "uses", "with", "working-directory"

(syntax-check)